fili-core/src/main/java/com/yahoo/bard/webservice/web/util/PaginationParameters.java

```diff
@@ -25,17 +25,17 @@
     private static final String FIRST = "first";
     private static final String LAST = "last";
 
-    private static final int DEFAULT_MAX_RESULTS_WITHOUT_FILTERS = 10000;
-    private static final int MAX_RESULTS_WITHOUT_FILTER = SYSTEM_CONFIG.getIntProperty(
-            SYSTEM_CONFIG.getPackageVariableName("max_results_without_filters"),
-            DEFAULT_MAX_RESULTS_WITHOUT_FILTERS
+    private static final int DEFAULT_MAX_RESULTS_PER_PAGE = 10000;
+    private static final int MAX_RESULTS_PER_PAGE = SYSTEM_CONFIG.getIntProperty(
+            SYSTEM_CONFIG.getPackageVariableName("max_results_per_page"),
```

Comment:
Changing this config param name is backwards incompatible. Can we continue to use the old param, logging a warning if it's still used, but respect it, while looking for the new one if it's there?

The order would be something like:

```groovy
MAX_RESULTS_PER_PAGE = max_results_without_filter 
    ? logErrorAndReturnMaxResultsWoFilterValue
    : max_results_per_page;
```